code/game/gamemodes/traitor/traitor.dm

```diff
@@ -345,6 +345,34 @@
 	if(M && M != traitor_mob)
 		to_chat(traitor_mob, "We have received credible reports that [M.real_name] might be willing to help our cause. If you need assistance, consider contacting them.")
 		traitor_mob.mind.store_memory("<b>Potential Collaborator</b>: [M.real_name]")
+		//let's also inform their contact that they might be called upon, but leave it vague.
+		inform_collab(M)

```

Comment:
Perhaps change this, so it runs in all game modes.
Then people can't use this to metagame the round type.